components/script/dom/element.rs

```diff
@@ -1718,6 +1718,27 @@ impl Element {
         self.set_attribute(local_name, AttrValue::String(value.into()));
     }
 
+    // Used for string attribute reflections where absence of the attribute returns null
+    pub fn get_nullable_string_attribute(&self, local_name: &LocalName) -> Option<DOMString> {
```

Comment:
Do you mind turning this comment into a proper rustdoc docstring? Please see https://doc.rust-lang.org/rust-by-example/meta/doc.html for how to do this.